source/routes/nlp.rst

```diff
@@ -0,0 +1,67 @@
+=======
++-------------------------+--------------------------+
+| Attribute               | Value                    |
++=========================+==========================+
+| Route Name              | nlp                      |                                    |
++-------------------------+--------------------------+
+| Route URI               | direct:nlp               |
++-------------------------+--------------------------+
+| Supported Operations(s) | Camel direct invocation  |
++-------------------------+--------------------------+
+
+Configuration
+=============
+
+The direct:nlp route expects the following LFH property settings:
+
+Exchange State
+--------------
++------------------------------+---------------------------------------------------------------------------+----------------------------+
+| LFH Properties               | Description                                                               | Default                    |
++==============================+===========================================================================+============================+
+| lfh.connect.nlp.uri          | NLP service request endpoint.                                             | Endpoint for ACD (us-east) |
++------------------------------+---------------------------------------------------------------------------+----------------------------+
+| lfh.connect.nlp.auth         | Camel http component authMethod, authUsername, and authPassword settings. |                            |
++------------------------------+---------------------------------------------------------------------------+----------------------------+
+| lfh.connect.nlp.dataformat   | Data format identifier for route (output Kafka topic prefix).             | NLP                        |
++------------------------------+---------------------------------------------------------------------------+----------------------------+
+| lfh.connect.nlp.messagetype  | Message type identifier for route (output Kafka topic suffix).            | RESONSE                    |
++------------------------------+---------------------------------------------------------------------------+----------------------------+
+| lfh.connect.nlp.request-json | Json request wrapper. "<REPLACE_TOKEN>" is replaced by the unstructured data at runtime. This is for nlp services that require application/json requests. If plain text is supported, this property may be left blank. | ACD json request wrapper. |
++------------------------------+---------------------------------------------------------------------------+----------------------------+
+
+Sample Annotator for Clinical Data (ACD) properties:
+----------------------------------------------------
+lfh.connect.nlp.uri=https://us-east.wh-acd.cloud.ibm.com/wh-acd/api/v1/analyze/wh_acd.ibm_clinical_insights_v1.0_standard_flow?version=2020-10-22
+lfh.connect.nlp.auth=&authMethod=Basic&authUsername=apikey&authPassword=ENC(<YOUR_ENCRYPTED_PASSWORD_HERE>)
```

Comment:
Are we able to utilize basic auth for the NLP route using the camel http4 component params?